components/CheckDocumentation/vacuum/MxidWraparound.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const MxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when multixact ID space utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once utilization exceeds <code>{critical}%</code>.
+      Resolves once utilization falls below the threshold.
+    </p>
+  );
+};
+
+const MxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl, serverVacuumFreezingUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        Multixact ID space utilization is high and approaching to wraparound.
+      </p>
+      <p>
+        This is about a multixact ID, not about the normal transaction ID. A
+        multixact ID is an internal identifier used support row locking by
+        multiple transactions. Multixact IDs are commonly used with{" "}
+        <code>SELECT ... FOR UPDATE</code> query.
+      </p>
+      <p>
+        Postgres runs autovacuums regularly and this helps keeping multixact ID
+        space utilization low by freezing old multixact IDs. There is also the
+        anti-wraparound autovacuum specific for freezing, when the regular
+        autovacuums are either not freezing or simply not running. This
```

Comment:
```suggestion
        autovacuums are either not freezing aggressively enough or simply not running. This
```
I think "not freezing" may make users wonder "why would they not be freezing" whereas "not freezing aggressively enough" suggests a configuration issue.